server.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const path = require('path')
+const nodemailer = require('nodemailer')
+const cors = require('cors')
+const { PORT = 4000 } = process.env
+
+const app = express()
+app.use(cors())
+app.use(express.json())
+
+app.use(express.static(path.join(__dirname, 'build')))
+
+app.get('/ping', function(req, res) {
+  return res.send('pong')
+})
+
+app.get('/', function(req, res) {
+  res.sendFile(path.join(__dirname, 'build', 'index.html'))
+})
+
+app.listen(PORT, () => console.log(`Server ready on http://localhost:${PORT}`))
+
+app.use(bodyParser.urlencoded({ extended: true }))
+
+app.get('/', (req, res) => {
```

Comment:
Is this route ever reached? I think the on line 18 will handle this request. Maybe you forgot to delete it?